[2023-06-13T05:53:18.589+0000] {processor.py:154} INFO - Started process (PID=17423) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:53:18.592+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:53:18.594+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:53:18.593+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:53:18.628+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:53:18.626+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:53:18.630+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:53:18.663+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.079 seconds
[2023-06-13T05:53:48.744+0000] {processor.py:154} INFO - Started process (PID=17455) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:53:48.746+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:53:48.748+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:53:48.748+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:53:48.760+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:53:48.758+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:53:48.761+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:53:48.780+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.039 seconds
[2023-06-13T05:54:19.416+0000] {processor.py:154} INFO - Started process (PID=17478) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:54:19.418+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:54:19.420+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:54:19.419+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:54:19.431+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:54:19.428+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:54:19.433+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:54:19.459+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.049 seconds
[2023-06-13T05:54:49.545+0000] {processor.py:154} INFO - Started process (PID=17510) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:54:49.548+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:54:49.549+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:54:49.549+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:54:49.560+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:54:49.557+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:54:49.561+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:54:49.581+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.040 seconds
[2023-06-13T05:55:20.265+0000] {processor.py:154} INFO - Started process (PID=17543) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:55:20.268+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:55:20.270+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:55:20.270+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:55:20.282+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:55:20.279+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:55:20.284+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:55:20.305+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.045 seconds
[2023-06-13T05:55:50.334+0000] {processor.py:154} INFO - Started process (PID=17573) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:55:50.338+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:55:50.339+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:55:50.339+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:55:50.349+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:55:50.347+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:55:50.351+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:55:50.375+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.044 seconds
[2023-06-13T05:56:21.047+0000] {processor.py:154} INFO - Started process (PID=17605) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:56:21.058+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:56:21.060+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:56:21.060+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:56:21.075+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:56:21.071+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:56:21.077+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:56:21.104+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.061 seconds
[2023-06-13T05:56:51.705+0000] {processor.py:154} INFO - Started process (PID=17628) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:56:51.709+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:56:51.711+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:56:51.711+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:56:51.724+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:56:51.720+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/groups/group_transforms.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_transforms.py", line 2, in <module>
    from airflow.operator.bash import BashOperator
ModuleNotFoundError: No module named 'airflow.operator'
[2023-06-13T05:56:51.725+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:56:51.752+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.053 seconds
[2023-06-13T05:57:15.776+0000] {processor.py:154} INFO - Started process (PID=17660) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:57:15.780+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:57:15.783+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:57:15.783+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:57:15.795+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:57:15.794+0000] {dagbag.py:319} INFO - File /opt/airflow/dags/groups/group_transforms.py assumed to contain no DAGs. Skipping.
[2023-06-13T05:57:15.797+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:57:15.820+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.050 seconds
[2023-06-13T05:57:45.873+0000] {processor.py:154} INFO - Started process (PID=17683) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:57:45.876+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:57:45.877+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:57:45.877+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:57:45.881+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:57:45.881+0000] {dagbag.py:319} INFO - File /opt/airflow/dags/groups/group_transforms.py assumed to contain no DAGs. Skipping.
[2023-06-13T05:57:45.882+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:57:45.901+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.031 seconds
[2023-06-13T05:58:16.731+0000] {processor.py:154} INFO - Started process (PID=17714) to work on /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:58:16.734+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/groups/group_transforms.py for tasks to queue
[2023-06-13T05:58:16.736+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:58:16.735+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:58:16.741+0000] {logging_mixin.py:120} INFO - [2023-06-13T05:58:16.741+0000] {dagbag.py:319} INFO - File /opt/airflow/dags/groups/group_transforms.py assumed to contain no DAGs. Skipping.
[2023-06-13T05:58:16.742+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_transforms.py
[2023-06-13T05:58:16.759+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/groups/group_transforms.py took 0.031 seconds
